flake8_trio.py

```diff
@@ -1146,6 +1147,52 @@ def visit_BoolOp(self, node: ast.BoolOp):
         self.uncheckpointed_statements = worst_case_shortcut
 
 
+_startable_targets: Dict[str, Tuple[str, ...]] = {
+    "trio": ("run_process", "serve_ssl_over_tcp", "serve_tcp", "serve_listeners"),
+    "": ("serve",),
+}
+
+
+class Visitor113(Flake8TrioVisitor):
+    def __init__(self, *args: Any, **kwargs: Any):
+        super().__init__(*args, **kwargs)
+        self.async_function = False
+        self.asynccontextmanager = False
+        self.aenter = False
+
+    def visit_AsyncFunctionDef(self, node: ast.AsyncFunctionDef):
+        outer = self.aenter
+        if any(
+            isinstance(n, ast.Name) and n.id == "asynccontextmanager"

```

Comment:
Probably also want to handle `<whatever>.asynccontextmanager` too; that'd usually but not always be `contextlib.`.